lambda/shared/utils/templates.ts

```diff
@@ -261,14 +261,10 @@ export const getEmailSubject = (messageType: EmailMessage, id = null) => {
 export const getInvitationEmail = (teamId: number, invitationLink: string) => `<h1>Hello Pathfinder SSO friend,</h1>
   <p>You are invited to join team #${teamId}</p>
   <p>
-    To accept this invitation, please follow this <a href="${API_URL}/teams/verify?token=${invitationLink}">link</a> to view the project in your dashboard</a>. This link will expire in <strong>2 business days.</strong>
+  Please follow this <a href="${API_URL}/teams/verify?token=${invitationLink}">link</a> to view your invitation in the dashboard. This link will expire in <strong>2 business days.</strong>
```

Comment:
Were you able to test this locally? I just wanted to confirm before I approve the PR

Reply:
actually because of my local db issue, I could not have a chance to test it locally. But all it needs is based on the previous work and apply a little bit changes to the text body, so I think should be no problem. But I would like to have it tested before approval, so it's a pending. Sorry for the confusion